package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "task_5_protractorjs",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "beforeStart": "webdriver-manager start",
+    "test": "protractor ."
```

Comment:
Please add a protractor config file, that has to be run.
The current `protractor .` script will result in an Error.

Reply:
fixed